src/installer/installer_test.go

```diff
@@ -400,6 +405,21 @@ var _ = Describe("installer HostRoleMaster role", func() {
 			ret := installerObj.InstallNode()
 			Expect(ret).Should(Equal(err))
 		})
+	})
```

Comment:
Did you intend to commit this?

Reply:
Yes, in the next line I created a new context `Context("Bootstrap role waiting for controller"...` that doesn't require the new `evaluateDiskSymlinkSuccess` mock
This way I could add the mock in all other relevant contexts instead of every test case.